pkg/api/apis/operators/v1alpha1/clusterserviceversion.go

```diff
@@ -133,6 +133,14 @@ func (c *ClusterServiceVersion) IsCopied() bool {
 	return false
 }
 
+func (c *ClusterServiceVersion) HasCAResources() bool {
+	// Return early if there are no owned APIServices
+	if len(c.Spec.APIServiceDefinitions.Owned)+len(c.Spec.WebhookDefinitions) == 0 {
```

Comment:
nit: `return !(len(c.Spec.APIServiceDefinitions.Owned)+len(c.Spec.WebhookDefinitions) == 0)` ?

Reply:
I don't believe that this suggestion makes the code more readable.